XMonad/Actions/CycleWorkspaceByScreen.hs

```diff
@@ -1,5 +1,3 @@
-
------------------------------------------------------------------------------
```

Comment:
Seems like this is removing a little too much sometimes :)

Reply:
Sometimes automation plays a bad joke on us. I am bringing these lines back.

Reply:
automation is bad just manually compile